### ISSUE TITLE
Certificate expired with mail-letsencrypt

### ISSUE THREAD

**Issue by supermar1010** [Mailu/Mailu#1288]
Hello,
I have the problem that the certs are expired that are required for SMTP... access.
I guessed they were auto renewed, but apparently this is not working for me. I restarted the container and also updated it. Any hints what to look for?

With best regards
Mario

**Comment by kaiyou** [comment]
We would need a bit more information to help you. First what TLS flavor are you using? And did you setup a specific reverse proxy in front of Mailu?

**Comment by supermar1010** [comment]
I'm using the TLS flavor: mail-letsencrypt, I'm having a reverse nginx in front of mailu, which does the certificate stuff for the website.
The ports for the website are just directly routed through the firewall.

**Comment by kaiyou** [comment]
Does the nginx proxy everything http to the front container?

If you have letsencrypt on that nginx, could it be possible that the `.well-known` path is served from there and never proxified to the front container? The mail-letsencrypt flavor still requires http traffic to be proxified to the front container that runs certbot.

**Comment by supermar1010** [comment]
Hmm no I don't think so I looked in all my site configs and the only one where I'm using the well known dir, is one of a completely different site with another url.
I'm generating the certs for the website with dns validation.
So it should be passed through? Or is there anything I need to tell the reverse-nginx, to route requests to .wellknown? 

This is my nginx config file:
```
server {
    listen 80;
    listen [::]:80;
    server_name ...;
    return 301 https://$host$request_uri;
}

server {
    listen 443 ssl;
    listen [::]:443 ssl;

    client_max_body_size 100m;

    server_name ...;

    location / {
        proxy_pass http://localhost:12345;
    }
    ssl_certificate /etc/letsencrypt/live/.../fullchain.pem;
    ssl_certificate_key /etc/letsencrypt/live/.../privkey.pem;
}
```

**Comment by kaiyou** [comment]
I did not spot anything obvious in the configuration. Could you provide some logs for the `front` container when it stats and runs the letsencrypt script?

**Comment by muhlemmer** [comment]
I have the feeling this is related to #1352.

**Comment by Nebukadneza** [comment]
Hi There,

The `Mailu`-Project is currently in a bit of a bind! We are short on man-power, and we need to judge if it is possible for us to put in some work on this issue.

To help with that, we are currently trying to find out which issues are actively keeping users from using `Mailu`, which issues have someone who want to work on them — and which issues may be less important. These a less important ones could be discarded for the time being, until the project is in a more stable and regular state once again.

In order for us to better assess this, it would be helpful if you could put a **reaction on this post** (use the :smiley: icon to the top-right).
- 👍️ if you **need this** to be able to use Mailu. Ideally, you’d also be able to test this on your installation, and provide feedback …
- 🎉 if you find it a **nice bonus**, but no deal-breaker
- 🚀 if you want to **work on it yourself**!
We want to keep this **voting open for 2 weeks** from now, so please help out!

**Comment by Nebukadneza** [comment]
Probably fixed in #1611 — please try the fix or wait for 1.8. Please do report back if this doesn’t fix the problem for you.
